docs/en/cloud/Alluxio-Presto-AMI.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: global
+title: Alluxio Presto Sandbox on AWS
+nickname: Alluxio Presto Sandbox
+group: Alluxio in the Cloud
+priority: 5
+---
+
+This guide describes how to launch a single node sandbox with Alluxio and Presto preconfigured to run queries.
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+The Alluxio-Presto sandbox is an Amazon Machine Image offered on Amazon EC2
+and features installations of MySQL, Hadoop, Hive, Presto, and Alluxio.
+
+The goal of this guide is to show how Alluxio can improve Presto’s query performance
+by reading through Alluxio to access locally cached data, originally stored in an Amazon S3 bucket.
+
+## Prerequisites
+
+- An account on [AWS](https://aws.amazon.com/)
+- Web console access to [EC2](https://console.aws.amazon.com/ec2)
+- A terminal window with [SSH](https://man.openbsd.org/ssh.1)
+
+## Launch an EC2 instance with Presto + Alluxio stack
+{% accordion launch %}
+  {% collapsible Open the web console %}
+Open a web browser and navigate to the [EC2 console](https://console.aws.amazon.com/ec2), logging in with your credentials when prompted.
+Note the AWS region in the upper right corner.
+
+![presto_sandbox_ec2_region]({{ '/img/presto_sandbox_ec2_region.png' | relativize_url }})
+
+Click on the region to open a dropdown of available regions.
+It is recommended to select the region that is geographically closest to your computer.
+
+On the left sidebar, click the `Instances` link to arrive at the following page: 
+
+![presto_sandbox_ec2_instances]({{ '/img/presto_sandbox_ec2_instances.png' | relativize_url }})
+
+Click the blue `Launch Instance` button.
+This will navigate you to the first of several steps to launch an instance.
+
+  {% endcollapsible %}
+  {% collapsible Choose an Amazon Machine Image (AMI) and Instance type %}
+In the left sidebar, there are 4 categories: Quick Start, My AMIs, AWS Marketplace, and Community AMIs.
+Click on `AWS Marketplace` and search for the Alluxio-Presto sandbox AMI by typing `alluxio-presto-sandbox` in the search bar.
+Exactly one result should appear; click on its blue `Select` button to proceed.
+
+![presto_sandbox_ec2_choose_ami]({{ '/img/presto_sandbox_ec2_choose_ami.png' | relativize_url }})
+
+Note that each AMI has a unique ID in the form `ami-xxxxxxxxxxxxxxxxx`.
+This ID differs depending on which AWS region was selected.
+If you would like to launch the AMI with spot instances, click on `Community AMIs`
+and among the 2 selections, choose the one with the shorter name.
```

Comment:
shorter name ... ❓ hahahahaha

Reply:
yes. this is the best way to describe it >.<